modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -9,11 +9,20 @@
 
 {product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
 
+[NOTE]
+====
+If you want to preview _enforcing_ the `restricted` profile on your non-production cluster as a Technology Preview, you can enable the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, which activates the `restricted` profile, pods are rejected if they violate pod security standards, instead of only logging a warning.
+
+Pod security admission restricted enforcement is only activated if you enable the `TechPreviewNoUpgrade` feature set after your {product-title} cluster is installed. It is not activated if you enable the `TechPreviewNoUpgrade` feature set during cluster installation.
+
+For information on enabling the `TechPreviewNoUpgrade` feature set, see _Understanding feature gates_.
```

Comment:
@bergerhoffer for **see _Understanding feature gates_.** is the link missing? or it is supposed to be just a text?

Others looks good to me.

Reply:
@gkarager The link isn't missing, it's supposed to just be text. The link is in the additional resources section below.

Unfortunately we can't add cross references (links to our own doc set) in our modules. :disappointed: So we have to do a workaround where we reference the section name and provide the actual link below the module.